portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,53 @@ function addRandomFact() {
 
   document.getElementById('img-container').src = image;
 }
+
+function nextExperience() {
+  const expContainer = document.getElementById('exp-container');
+  const exps = [
+    {title: 'WORK EXPERIENCE:', dets: ''}, 
+    {title: 'STEP INTERN AT GOOGLE', dets: 'May 2020-Present'}, 
+    {title: 'RESEARCH ASSISTANT AT LMU', dets: 'Sep 2019-Mar 2019'},
+    {title: 'SUMMER INTERN AT ELECTRO SCAN INC.', dets: 'Jun 2018-Aug 2018'},
+    {title: 'SUMMER LIFEGUARD AT YMCA', dets: 'May 2017-Jun 2017'},
+    {title: 'EDUCATION:', dets: ''},
+    {title: 'LOYOLA MARYMOUNT UNIVERSITY', dets: ' Pursuing Civil Engineering, Computer Science'},
+    {title: 'SACRAMENTO COUNTRY DAY SCHOOL', dets: 'High School Diploma (2019)'}
```

Comment:
`dets` is not a commonly used abbreviation in programming (there's only a few abbreviations that are commonplace enough to be used like this). It would be better to go with the full `details` name here instead.

Reply:
good to know!